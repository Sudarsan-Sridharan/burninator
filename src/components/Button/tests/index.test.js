import React from 'react';
import { render } from 'testing-utils';

import { BasicButton } from '../';

describe('<Button/>', () => {
  test('snapshot', () => {
    const { container } = render(<BasicButton>123</BasicButton>);
    expect(container.firstChild).toMatchSnapshot();
  });
});
