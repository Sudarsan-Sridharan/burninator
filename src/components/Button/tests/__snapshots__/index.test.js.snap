// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button/> snapshot 1`] = `
<button
  class="vs-btn-d vs-outline-secondary-btn"
  type="button"
>
  123
</button>
`;
